src/components/contact-info/package.json

```diff
@@ -6,10 +6,9 @@
   },
   "dependencies": {
     "@bcrs-shared-components/interfaces": "^2.0.0",
-    "vue": "^2.7.14",
     "vue-the-mask": "^0.11.1"
```

Comment:
Since you import something from "vue", you should include it as a dependency here.

Looks OK to merge otherwise 👍